app/presenters/queries/link_set_presenter.rb

```diff
@@ -14,7 +14,9 @@ def initialize(link_set, version)
       def present
         link_set.as_json
```

Comment:
You have to check this, but I think we need a `includes(:links)` here now to avoid a N+1 query.